pool/_TEST_pool_native_swap_test.gn_OK

```diff
@@ -37,19 +36,19 @@ func TestMint(t *testing.T) {
 
 	// prepare ugnot
 	testBanker := std.GetBanker(std.BankerTypeRealmIssue)
-	testBanker.IssueCoin(test1, "ugnot", 804_031)
+	testBanker.IssueCoin(test1, "ugnot", 804_032)
 
 	// simulate transfer & decrase
-	std.TestSetOrigSend(std.Coins{{"ugnot", 804_031}}, nil)
-	testBanker.RemoveCoin(std.GetOrigCaller(), "ugnot", 804_031)
+	std.TestSetOrigSend(std.Coins{{"ugnot", 804_032}}, nil)
+	testBanker.RemoveCoin(std.GetOrigCaller(), "ugnot", 804_032)
 
 	// Deposit(wrap)
 	std.TestSetPrevAddr(test1)
 	wugnot.Deposit()
 
 	std.TestSetPrevAddr(test1)
-	foo.Approve(a2u(consts.POOL_ADDR), 295801)
-	wugnot.Approve(a2u(consts.POOL_ADDR), 804031)
+	foo.Approve(a2u(consts.POOL_ADDR), 295802)
```

Comment:
Are these differences in the approvals due to some rounding errors?

Reply:
`bigint` really didn't round up anything. As matter of fact it didn't support rounding.
However with u256, i256 we can round up things.